instance.tf

```
@@ -27,6 +27,16 @@ resource "google_sql_database_instance" "instance" {
     ip_configuration {
       ipv4_enabled = true
       require_ssl  = true
+
+      dynamic "authorized_networks" {
+        for_each = var.authorized_networks
+        iterator = network
+
+        content {
+          name  = network.key
+          value = network.value
```

CIDR makes for a good unique key, while names do not
```suggestion
          name  = network.value
          value = network.key
```